extensions/wikia/WikiaMobile/js/topbar.js

```diff
@@ -18,6 +18,7 @@ function (
 
 	var d = w.document,
 		wkPrfTgl = d.getElementById('wkPrfTgl'),
+		$wkPrfTgl = $(wkPrfTgl),
```

Comment:
I know it was already there, but those variable names doesn't mean anything to me. Can we improve it here?